homeassistant/bootstrap.py

```diff
@@ -157,6 +158,12 @@ def from_config_dict(config: Dict[str, Any],
 
     await hass.async_block_till_done()
 
+    # Kick off loading the registries. They don't need to be awaited.
+    asyncio.gather(
```

Comment:
We don't need to await the gather here, gather starts the tasks immediately and if anything tries to access the registries, it will be blocked until the registries have finished loading. (I've tested this). I don't use `hass.async_create_task` because that would add the tasks to the tracked task list, and therefore would end up getting waited for in the `block_till_done below.